docs/reference/files.md

```diff
@@ -313,6 +325,8 @@ Any additional metadata Directus was able to scrape from the file. For images, t
 	"filesize": 3442252,
 	"width": 3456,
 	"height": 5184,
+	"focal_point_x": 1598,
+	"focal_point_y": 1827,
```

Comment:
As far as I know theres no industry standard meta field for focal points so they'll _technically_ be `null` upon initialization :pray: 

```suggestion
	"focal_point_x": null,
	"focal_point_y": null,
```